### ISSUE TITLE
Apk needs a "Network Access" spoof

### ISSUE THREAD

**Issue by paceast** [Genymobile/gnirehtet#422]
It says no network available when I turn off mobile data, disconnected from a wifi hotspot nor entered airplane mode.

**Comment by rom1v** [comment]
Related: https://github.com/Genymobile/gnirehtet/issues/37#issuecomment-317695253

**Comment by paceast** [comment]
You're right. I'm sorry for duplicating this. Do you have a solution for this?

**Comment by LeeBinder** [comment]
@altern00b I habven't tested this myself yet, but accdg. to evolu1 only with root via Magisk + LSPosed (newer) or (older) EdXposed (that's what he used), see https://github.com/Genymobile/gnirehtet/issues/397#issuecomment-872851962
